### PR TITLE
Add further shields and move images to repository.

### PR DESCRIPTION
**Pull request by Landrash**
Adds logo to top of page. (Larger logo might be appropriate but repurposed the one in the repo)
Adds a few more shields.
Creates copies of the images in the repository.

Signed-off-by: Fredrik Lindqvist <landrash@mail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,18 @@
 # ZeroTier-GUI
+<img src="zerotier-gui.png">
 
+[![GitHub release (latest SemVer)](https://img.shields.io/github/v/release/tralph3/ZeroTier-GUI)](https://github.com/tralph3/ZeroTier-GUI/releases)
 [![License: GPL v3](https://img.shields.io/badge/License-GPL%20v3-blue.svg)](https://github.com/tralph3/ZeroTier-GUI/blob/master/LICENSE)
+[![Send me a small gift](https://img.shields.io/badge/send%20me%20a%20small%20gift-paypal-blue.svg)](https://paypal.me/tralph3)
```
Comment (tralph3) [inline README.md]: Neat idea about the badges! However I saw that there's a flat version of these that I like better aesthetically. I'd say replacing these with those is better.

Comment (tralph3) [reply README.md]: Just adding `?style=flat-square` to the end of the image sources is enough.

**Comment by Landrash** [comment]
Seems like a good idea :)

**Comment by tralph3** [comment]
Neat ^^ I'm gonna merge it then.

**Pull request merged**